site/themes/getenvoy/static/css/main.css

```diff
@@ -2751,22 +2749,22 @@ article.repo > ul > li {
 		text-shadow: 0 0 1px rgba(0,0,0,.25);
 	}
 
-	code button{
+	pre button{
 		cursor: pointer;
 		position: absolute;
-                right: 8px;
-		top: 8px;
+    right: 12px;
```

Comment:
super nit, can we adjust the tabbing here, so the indentation format stays as before?

Reply:
arghh editorconfig problem :-P